modules/client/src/internal/client/methods.ts

```diff
@@ -1198,11 +1194,11 @@ export class ClientMethods extends ClientCore implements IClientMethods {
         ],
       };
     }
-  
+
     let daoActions = toDaoActions(iproposal.actions);
     const classifiedActions = classifyProposalActions(daoActions);
 
-    // remove upgradeToAndCall action 
+    // remove upgradeToAndCall action
     if (containsDaoUpdateAction(classifiedActions)) {
```

Comment:
For clarity, it would be good to rename `containsDaoUpdateAction` to `startsWithDaoUpdateAction`